win32/updateBuildLibs.py

```diff
@@ -265,6 +251,9 @@ def main():
         usage()
 
     print('Updating source by %s branch.' % branch)
+    if not os.path.exists(LOG_PATH):
+        os.makedirs(LOG_PATH)
+
```

Comment:
Sorry, I misread the code, and did not notice you had moved the log creation to the main method. I needed to expand the code in the diffs.